### DISCUSSION TITLE
Strange graphs and texts (even after upgrade)

### DISCUSSION THREAD

**Discussion by skogizzz** [UtilitechAS/amsreader-firmware#516] [Q&A] [answered]
I installed the Pow-K+ 3 days ago and it's a very simple setup.

But I have 2 problems.
First one is that I don't get any graphs. I have tried to zero out the cfg but the readings comes back, maybe I'm doing it wrong?
Second thing. I upgraded to 2.2.12 and in the upper right (Real time calculation) the order of the text seems strange.

Any help here would be good :)

![image](https://user-images.githubusercontent.com/129111486/230001710-bf363651-7be3-4440-871f-3794ceb28974.png)

**Comment by gskjold** [comment]
What country? Does the kwh reading on the dashboard match what is on your meter? I am wondering if maybe this needs a kwh multiplier in config

**Reply by skogizzz** [reply]
This is Sweden.

Right now I have 0.1 in the config for kWh
![image](https://user-images.githubusercontent.com/129111486/230015130-fd5b78d3-bb92-4570-8c6d-3255a1b091e7.png)

**Reply by gskjold** [reply]
OK, if you only have this high peak on the day graph, I think we should wait until it passes and see if it shows up again

**Reply by skogizzz** [reply]
Now it looks like right values shows up but how come that it round of the value? It was 8,84 kWh in the realtime stats window (upper right) before it showed 8.0 with the graphs.

Can I modify how it will round of values or is it that the older faulty value with very high value makes it round of the value?

![image](https://user-images.githubusercontent.com/129111486/230084103-a82287c4-d44a-4583-82f3-0dc51a87d0fe.png)

**Reply by ArnieO** [reply]
I think this is just due to the scaling of the graph, as the high peak is still there. 

Try this:
Download the config file (from info page), and edit away the peak as explained here: https://amsleser.no/blog/post/28-correcting-stored-values-by-editing-the-configuration-file
Then upload the config file back to the device. This should remove the peak and restore a normal graph.

**Comment by ArnieO** [comment]
A really weird thing here is that the formatting issue of the "Real time consumption" tile also shows up, ref https://github.com/UtilitechAS/amsreader-firmware/issues/514 - which I thought was an issue related to ESP8266.

Did you try a Factory Reset? (Button on bottom of the Config page)

**Reply by skogizzz** [reply]
I can try a factory reset when I'm back home. 
Backup the config-file before reset I assume?

**Reply by ArnieO** [reply]
Yes, it will save you from having to re-enter the various settings you made manually.

**Comment by ArnieO** [comment]
Does it show accumulated meter reading after it has passed whole hour?
![image](https://user-images.githubusercontent.com/10295178/230008002-6c884e76-5bf0-4103-bf07-3be8921aeb78.png)
The value shown here should be the same as what you can read on the LCD screen of your meter. If not, you'll need to set a kWh multiplier.
We have seen before that some few Kamstrup meters in Sweden report an extra digit - and we have not found any way of detecting that automatically.

**Reply by skogizzz** [reply, marked as answer]
Yes now after I changed to multipier 0.1 in the kWh box it's correct value
![image](https://user-images.githubusercontent.com/129111486/230015697-b382bddb-f7f6-443d-863f-9d623a8cab28.png)
